src/crispy/stdfs.h

```diff
@@ -17,7 +17,7 @@
     #include <filesystem>
     #include <system_error>
 namespace FileSystem = std::filesystem;
-using FileSystemError = std::error_code;
+using file_system_error = std::error_code;
```

Comment:
as mentioned above. I think in a future PR we can finally completely get rid of the last bits that were needed when we still partially used Boost if std::filesystem wasn't present.

Reply:
Will create issue